src/features/filter.ts

```diff
@@ -1,12 +1,78 @@
+import { Filters } from '../types';
```

Comment:
It was not required in this task, but as a general rule of thumb I'd prefer having different folder structure for the project. 
IDK why is it recommended to put this type of code into `features` folder, I believe this approach is not the best example of folder structure. 

```
// actions 
   => filters
// reducers 
  = > filters 
```